src/server/game/AI/SmartScripts/SmartScriptMgr.cpp

```diff
@@ -1019,6 +1028,18 @@ bool SmartAIMgr::IsEventValid(SmartScriptHolder& e)
                     return false;
                 }
                 break;
+            case SMART_EVENT_QUEST_OBJ_COPLETETION:
+                //    if (!sObjectMgr->GetQuestObjective(e.event.questObjective.id))
```

Comment:
In general, the idea is not copy and paste, if you don't need some code, avoid to comment, else please remove.

Reply:
I will comment it because to implement it later it is necessary to implement the QuestObjectives first, I left it commented because soon I will try to implement the QuestObjectives.

Reply:
Oh, I understand, so it should be better if you put a //TODO: inside the section, to keep in mind that.

Reply:
fix :)